app/controllers/concerns/spotlight/base.rb

```diff
@@ -43,6 +42,16 @@ def autocomplete_json_response_for_document(doc)
         iiif_manifest: doc[Spotlight::Engine.config.iiif_manifest_field]
       }
     end
-    # rubocop:enable Metrics/AbcSize
+
+    # Some pipeline steps may return an array instead of a string
+    # to_str throws an error if the object is not a string
+    # This workaround does a join on the array to allow values to be returned
+
+    private
+
+    def autocomplete_title(heading)
```

Comment:
I think it would be clearer to keep the comment with the `autocomplete_title` method rather than the `private` keyword in case other methods are added later. What do you think?
```suggestion
    private

    # Some pipeline steps may return an array instead of a string
    # to_str throws an error if the object is not a string
    # This workaround does a join on the array to allow values to be returned
    def autocomplete_title(heading)
```

Reply:
I had it that way and then moved it around. The suggestion sounds good! I'll look at setting up a test. thanks!